Cargo.toml

```diff
@@ -12,16 +12,16 @@ authors = ["Sean McArthur <sean.monstar@gmail.com>",
 keywords = ["http", "hyper", "hyperium"]
 
 [dependencies]
-cookie = "*"
 httparse = "*"
 log = ">= 0.2.0"
-mime = "*"
-openssl = "*"
-rustc-serialize = "*"
 time = "*"
-unicase = "*"
 url = "*"
 
+[dependencies.hypernet]
+path = "hypernet"
```

Comment:
This is probably true.

If we split hyper consequently there will be no "hyper" crate only "hyperclient" and "hyperserver". I have no idea what the current crate will be useful for.